internal/config/config.go

```diff
@@ -126,6 +127,17 @@ func (conf *Config) HasConfiguredOrganization(slug string) bool {
 	return ok
 }
 
+func (conf *Config) SavePipelines(pipelines []pipeline.Pipeline) error {
+	names := make([]string, len(pipelines))
+	for i, n := range pipelines {
+		names[i] = n.Name
+	}
+
+	conf.localConfig.Set("pipelines", names)
+
+	return conf.localConfig.WriteConfig()
```

Comment:
We can `return` early from this by checking the `len > 0` at the beginning of the function, it will also avoid setting a `pipelines` key where not needed.

Reply:
Oh thats good point. In that case it would write an empty list to the config which would resolve first and then never go out to the API to load more either so that makes sense to not "cache" an empty list I think 👍

Reply:
I think I'll close this PR. And I've applied this fix in #249